api/lib/source/polkadotV0-source.js

```diff
@@ -529,6 +529,207 @@ class polkadotAPI {
       }
     )
   }
+
+  constructProposal = function (api, bytes) {
```

Comment:
weird syntax
```suggestion
  constructProposal(api, bytes) {
```

Reply:
yeah, coming straight from the polkadotjs repo :sweat_smile: